projects/packages/videopress/src/client/block-editor/blocks/video/edit.native.tsx

```diff
@@ -115,12 +127,13 @@ export default function VideoPressEdit( {
 
 				// Delete attributes that are not needed.
 				delete newBlockAttributes.poster;
+				delete newBlockAttributes.src;
 
 				setIsReplacingFile( { isReplacing: false, prevAttrs: {} } );
 				replaceBlock( clientId, createBlock( 'videopress/video', newBlockAttributes ) );
 				return;
 			}
-			setAttributes( { id: newVideoData.id, guid: newVideoData.guid } );
+			setAttributes( { id: newVideoData.id, guid: newVideoData.guid, src: undefined } );
```

Comment:
We are using the local file URL in the `src` attribute, hence in order to avoid persisting the local URL, we remove the attribute when the upload finishes.